mineinabyss-systems/src/main/kotlin/com/mineinabyss/pvp/survival/SurvivalPvpFeature.kt

```diff
@@ -0,0 +1,22 @@
+package com.mineinabyss.pvp.survival
+
+import com.mineinabyss.idofront.commands.execution.ExperimentalCommandDSL
+import com.mineinabyss.idofront.plugin.registerEvents
+import com.mineinabyss.mineinabyss.core.AbyssFeature
+import com.mineinabyss.mineinabyss.core.MineInAbyssPlugin
+import com.mineinabyss.mineinabyss.core.geary
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("survival_pvp")
+@ExperimentalCommandDSL
+class SurvivalPvpFeature: AbyssFeature {
+    override fun MineInAbyssPlugin.enableFeature() {
+        registerEvents(SurvivalPvpListener())
+
+        geary {
+            SurvivalPvpCommandExecutor()
```

Comment:
You don't need to put this into a `geary` block. Id prefer you also just use the `commands` block and put commands right in here.